src/libraries/System.Drawing.Common/src/System/Drawing/macFunctions.cs

```diff
@@ -28,11 +28,9 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
-#undef DEBUG_CLIPPING
```

Comment:
FYI - note to reviewers, this brings the changes from https://github.com/dotnet/runtime/pull/54245 here. This is necessary in order to test the rest of the CI legs to ensure the other changes are valid.

This PR shouldn't be merged until https://github.com/dotnet/runtime/pull/54245 is merged.